soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1365,8 +1365,8 @@ std::map<RandomizerCheck, RandomizerInf> rcToRandomizerInf = {
     { RC_FOREST_TEMPLE_MQ_LOBBY_POT_4,                                  RAND_INF_FOREST_TEMPLE_MQ_LOBBY_POT_4 },
     { RC_FOREST_TEMPLE_MQ_LOBBY_POT_5,                                  RAND_INF_FOREST_TEMPLE_MQ_LOBBY_POT_5 },
     { RC_FOREST_TEMPLE_MQ_LOBBY_POT_6,                                  RAND_INF_FOREST_TEMPLE_MQ_LOBBY_POT_6 },
-    { RC_FOREST_TEMPLE_MQ_LOWER_STALFOS_POT_1,                          RAND_INF_FOREST_TEMPLE_MQ_LOWER_STALFOS_POT_1 },
-    { RC_FOREST_TEMPLE_MQ_LOWER_STALFOS_POT_2,                          RAND_INF_FOREST_TEMPLE_MQ_LOWER_STALFOS_POT_2 },
+    { RC_FOREST_TEMPLE_MQ_WOLFOS_POT_1,                          RAND_INF_FOREST_TEMPLE_MQ_LOWER_STALFOS_POT_1 },
+    { RC_FOREST_TEMPLE_MQ_WOLFOS_POT_2,                          RAND_INF_FOREST_TEMPLE_MQ_LOWER_STALFOS_POT_2 },
     { RC_FOREST_TEMPLE_MQ_UPPER_STALFOS_POT_1,                          RAND_INF_FOREST_TEMPLE_MQ_UPPER_STALFOS_POT_1 },
```

Comment:
Is the spacing here just different in github or also in the IDE? Otherwise it'd be nice to have them aligned again. This goes for all of the lines in this file.

Reply:
Oops, forgot this one, will clean it up when I clean up at some point